ai/core/src/unitycatalog/ai/core/databricks.py

```diff
@@ -704,6 +731,29 @@ def _execute_uc_functions_with_serverless(
             error = f"Failed to execute function with command `{sql_command_msg}`\nError: {e}"
             return FunctionExecutionResult(error=error)
 
+    @retry_on_session_expiration
+    def _execute_uc_functions_with_local(
+        self, function_info: "FunctionInfo", parameters: Dict[str, Any]
+    ) -> FunctionExecutionResult:
+        if not is_scalar(function_info):
+            raise ValueError(
+                "Local sandbox execution is only supported for scalar Python functions. "
+                "Use 'serverless' execution mode for table functions."
+            )
+        _logger.info("Using local sandbox to execute functions.")
+
+        parameters = process_function_parameter_defaults(function_info, parameters)
+        python_def = get_callable_definition(function_info)
+        try:
+            succeeded, result = run_in_sandbox_subprocess(python_def, parameters)
```

Comment:
Why doesn't `run_in_sandbox` work for databricks?

Reply:
It's a long explanation - let's discuss offline :)